lib/Controller/PageController.php

```diff
@@ -288,9 +308,11 @@ protected function guestEnterRoom(string $token, string $password): Response {
 			if ($token) {
 				$redirectUrl = $this->url->linkToRoute('spreed.Page.showCall', ['token' => $token]);
 			}
-			return new RedirectResponse($this->url->linkToRoute('core.login.showLoginForm', [
+			$response = new RedirectResponse($this->url->linkToRoute('core.login.showLoginForm', [
 				'redirect_url' => $redirectUrl,
 			]));
+			$response->throttle();
```

Comment:
I have just realized that this throttles guests entering in a public call, even if the token is right. I guess that is not intended, is it? :-)

Reply:
Ignore that, I got confused by the indentation GitHub :facepalm:

The throttling is expected when opening a conversation if you are already throttled for conversation tokens (for example, from testing other endpoints...) due to the annotation in [showCall](https://github.com/nextcloud/spreed/pull/7504/files#diff-70e5309fcd3311d771b3db9c93490ea270fd5894769093765ec37edb68e5dd9bR122).